src/control-plane/auth/cognito-auth.ts

```diff
@@ -255,17 +265,22 @@ export class CognitoAuth extends Construct implements IAuth {
       customAttributes: {
         userRole: new cognito.StringAttribute({ mutable: true, minLen: 1, maxLen: 256 }),
       },
-      advancedSecurityMode:
-        props?.enableAdvancedSecurityMode == false
```

Comment:
are we still taking the right action based on whether this value `enableAdvancedSecurityMode` is false?